bundles/binding/org.openhab.binding.nibeheatpump/src/main/resources/Nibe F750 variable information.csv

```diff
@@ -0,0 +1,270 @@
+"Room sensor setpoint S1";"Sets the room temperature setpoint for the system";47398;"�C";s16;10;50;300;200;R/W;
+"Room sensor factor S4";"Setting of how much the difference between set and actual room temperature should affect the supply temperature.";47399;"";u8;10;0;60;20;R/W;
+"Room sensor factor S3";"Setting of how much the difference between set and actual room temperature should affect the supply temperature.";47400;"";u8;10;0;60;20;R/W;
+"Room sensor factor S2";"Setting of how much the difference between set and actual room temperature should affect the supply temperature.";47401;"";u8;10;0;60;20;R/W;
+"Room sensor factor S1";"Setting of how much the difference between set and actual room temperature should affect the supply temperature.";47402;"";u8;10;0;60;20;R/W;
+"Speed circ.pump Pool";"";47415;"%";u8;1;0;100;70;R/W;
+"Speed circ.pump Cooling";"";47417;"%";u8;1;0;100;70;R/W;
+"preset flow clim. sys.";"Preset flow setting for climate system. 0 = manual setting, 1 = radiator, 2 = floor heating, 3 = radiator + floor heating.";47442;"";u8;1;0;3;1;R/W;
+"Max time defrost";"Maximum duration of the defrost";47473;"min";u8;1;0;0;0;R/W;
+"Night cooling";"If the fan should have a higher speed when there is a high room temp and a low outdoor temp. 0=Off 1=On";47537;"";u8;1;0;1;0;R/W;
+"Start room temp. night cooling";"";47538;"�C";u8;1;20;30;25;R/W;
+"Night Cooling Min. diff.";"Minimum difference between room temp and outdoor temp to start night cooling";47539;"�C";u8;1;3;10;6;R/W;
+"DEW accessory";"Activates the DEW accessory";47555;"";u8;1;0;1;0;R/W;
+"Operational mode";"The operational mode of the heat pump 0=Auto 1=Manual 2=Add. heat only";47570;"";u8;1;0;0;0;R/W;
+"Operational mode charge pump";"";48134;"";u8;1;10;20;20;R/W;
+"SAM supply air curve: outdoor temp T3";"The supply air curve is defined by 3 supply air temperatures at 3 different outdoor temperatures T1, T2 and T3.";48158;"�C";s16;10;-400;200;150;R/W;
+"SAM supply air curve: outdoor temp T2";"The supply air curve is defined by 3 supply air temperatures at 3 different outdoor temperatures T1, T2 and T3.";48159;"�C";s16;10;-390;190;0;R/W;
+"SAM supply air curve: outdoor temp T1";"The supply air curve is defined by 3 supply air temperatures at 3 different outdoor temperatures T1, T2 and T3.";48160;"�C";s16;10;-400;200;-150;R/W;
+"SAM supply air curve: supply air temp at T3";"The supply air curve is defined by 3 supply air temperatures at 3 different outdoor temperatures T1, T2 and T3.";48161;"�C";s16;10;160;520;220;R/W;
+"SAM supply air curve: supply air temp at T2";"The supply air curve is defined by 3 supply air temperatures at 3 different outdoor temperatures T1, T2 and T3.";48162;"�C";s16;10;160;520;220;R/W;
+"SAM supply air curve: supply air temp at T1";"The supply air curve is defined by 3 supply air temperatures at 3 different outdoor temperatures T1, T2 and T3.";48163;"�C";s16;10;160;520;220;R/W;
+"SCA accessory";"Activates the SCA accessory";48201;"";u8;1;0;1;0;R/W;
```

Comment:
this file seem to have some special character issues … "°C" seem to be checked in correctly